src/main/java/com/mixpanel/android/mpmetrics/MixpanelFCMMessagingService.java

```diff
@@ -243,13 +255,24 @@ protected static Notification buildNotification(Context context, Intent inboundI
         maybeSetCustomIconColor(builder, data);
         maybeAddActionButtons(builder, data, context);
         maybeSetChannel(builder, data, context);
+        maybeSetSubTitle(builder, data, context);
+        maybeSetExpandableImage(builder, data, context);
+        maybeSetIconImage(builder, data, context);
```

Comment:
This just slams the changes of `maybeSetLargeIcon`. Let's merge those functions and call it `maybeSetThumbnailImage`